src/components/Map/__tests__/Map.spec.tsx

```diff
@@ -1,5 +1,8 @@
 import * as React from 'react';
-import toJson from 'enzyme-to-json';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
is there no way to remedy this?

Reply:
I don't know. It was here, i'll check it

Reply:
I can off this rule. But as i remember it's not good solution

Reply:
Sorry, I don't get your answer. Why do we need this lint exception here?

Reply:
Removed